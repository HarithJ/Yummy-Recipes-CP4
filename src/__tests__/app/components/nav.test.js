import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Nav from '../../../components/auth/nav.js'

describe("nav component", () => {
  const nav = shallow(<Nav link={"register"} linkTitle={"Register"}/>)

  it("renders properly", () => {
    expect(shallowToJson(nav)).toMatchSnapshot();
  })
})
