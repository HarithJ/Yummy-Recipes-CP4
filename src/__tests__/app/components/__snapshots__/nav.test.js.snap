// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav component renders properly 1`] = `
<nav
  className="navbar navbar-expand-sm navbar-dark bg-dark"
>
  <button
    aria-controls="navbarSupportedContent"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarSupportedContent"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    className="collapse navbar-collapse"
    id="navbarSupportedContent"
  >
    <ul
      className="navbar-nav mr-auto"
    >
      <li
        className="nav-item"
      >
        <Link
          className="nav-link"
          replace={false}
          to="register"
        >
          Register
        </Link>
      </li>
    </ul>
  </div>
</nav>
`;
